src/Sylius/Component/Core/Model/Order.php

```diff
@@ -477,12 +477,12 @@ public function getNonDiscountedItemsTotal(): int
         return $total;
     }
 
-    public function getByGuest(): bool
+    public function getCreatedByGuest(): bool
```

Comment:
Shouldn't it be `isCreatedByGuest` instead ?

Reply:
It probably should... but this is a PR just to fix conflicts after upmerge, we can fix it from 1.10 ⬆️ later on :)

Reply:
It could be, it looks probably more natural, but it should be changed on 1.10 branch, as the new field has been introduced there, in this PR I'm only fixing the problem after upmerge branches